server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ResetDatabase.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Free & Fair Colorado RLA System
+ * 
+ * @title ColoradoRLA
+ * @created Aug 12, 2017
+ * @copyright 2017 Free & Fair
+ * @license GNU General Public License 3.0
+ * @creator Joe Kiniry <kiniry@freeandfair.us>
+ * @description A system to assist in conducting statewide
+ * risk-limiting audits.
+ */
+
+package us.freeandfair.corla.endpoint;
+
+import spark.Request;
+import spark.Response;
+
+import us.freeandfair.corla.asm.ASMEvent;
+import us.freeandfair.corla.asm.ASMEvent.DoSDashboardEvent;
+import us.freeandfair.corla.asm.AbstractStateMachine;
+import us.freeandfair.corla.asm.AuditBoardDashboardASM;
+import us.freeandfair.corla.asm.CountyDashboardASM;
+import us.freeandfair.corla.asm.DoSDashboardASM;
+import us.freeandfair.corla.asm.PersistentASMState;
+import us.freeandfair.corla.model.BallotManifestInfo;
+import us.freeandfair.corla.model.CastVoteRecord;
+import us.freeandfair.corla.model.Contest;
+import us.freeandfair.corla.model.County;
+import us.freeandfair.corla.model.CountyDashboard;
+import us.freeandfair.corla.model.DoSDashboard;
+import us.freeandfair.corla.model.Elector;
+import us.freeandfair.corla.model.UploadedFile;
+import us.freeandfair.corla.persistence.Persistence;
+import us.freeandfair.corla.query.PersistentASMStateQueries;
+
+/**
+ * Reset the database, except for authentication information.
```

Comment:
Note that it _does_ remove all _records_ about uploaded artifacts; just not the artifact blobs themselves.